src/client/components/app-core/variables.css

```diff
@@ -10,6 +10,8 @@
   --background-color: #fff;
   --text-color: #4c4c4c;
   --text-light-color: #adadad;
+  --focus-color: #42BF9A;
```

Comment:
nitpick, but make the letters lowercase: `42bf9a`

Reply:
Changed this.